app/views/videos/new.html.erb

```diff
@@ -37,8 +37,6 @@
     </p>
   </div>
 </center>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"
```

Comment:
@rt4914 we cannot remove the jQuery from the videos page without making other changes, I can see a usage of jQuery just below https://github.com/forem/forem/blob/76848fa9373231b4174fa910a05acb112566a1a5/app/views/videos/new.html.erb#L45-L48
which will break if we remove jQuery. We'd need to check if there is more like this as well in the other JS files.

Reply:
Sorry I know this PR is still in Draft, but I thought I'd bring it to your attention earlier rather than later.

Reply:
Thanks a lot @Ridhwana for this. 
I am actually trying to find the feature where I can actually use this video-upload functionality but unable to find it.

Any idea how to use this feature?

Reply:
Update on above query -- enabled the video setting so as to reach the page which uses jQuery.
<img width="895" alt="Screenshot 2023-01-18 at 4 32 25 PM" src="https://user-images.githubusercontent.com/9396084/213155169-ede1afa9-3f84-4192-a648-e45891c16919.png">

Reply:
@Ridhwana I think we cannot remove jQuery. Have a look at this video https://loom.com/share/8786eabfc98f412a9b34aeb367d46a86

Currently we are using `additional_data` [usage](https://github.com/forem/forem/blob/88d9eef916fb80736df3307f034faa4549050a14/app/views/videos/new.html.erb#L46) option which does need jQuery as per https://github.com/waynehoover/s3_direct_upload/blob/master/README.md#options-for-s3upload-jquery-plugin

Reply:
I have tried to find any Javascript/Rails based plugin which does not use jQuery for S3 direct upload but so far no luck in that.

Reply:
@rt4914 I think its okay for now, in that case let's keep the jQuery around for the video upload, it's scoped to that page load so I think its okay and we can let @mirie know and perhaps we can make a note of it for the future.  

In the meantime its a big win that we'll be removing it from the admin.html.erb which loads jQuery on all the admin pages. 

Thnak you so much for your effort investigating this!

Reply:
@Ridhwana Thanks a lot for you guidance. I have removed jquery from admins and updated the code too.

We cannot remove it from app/views/videos/new.html.erb as its is dependent on thrid-party : https://github.com/waynehoover/s3_direct_upload/blob/master/README.md#options-for-s3upload-jquery-plugin